assets/styles/main.css

```diff
@@ -32,7 +43,32 @@ header > nav {
   padding: 0;
 }
 
-header > nav a.current {
+header > nav a.available {
```

Comment:
This was probably from your global find/replace. Change it back to current so the top nav link styling isn't broken.